src/App.js

```diff
@@ -54,10 +59,11 @@ class App extends Component {
         </BrowserRouter>
       );
     }
-    if (user.is_admin === true) {
+    if (user.is_admin) {
```

Comment:
Tried to clean code: 'if (user.is_admin === true) is the same as ' (if (user.is_admin))'

Reply:
thanks